doc/docs/reference/patient.md

```diff
@@ -11,3 +11,9 @@ Returns a new `Episode` for this patient.
 Return the dictionary representation of this patient - suitable for serialization.
 
     patient.to_dict(user)
+
+
+#### bulk_update
+pass in a dictionary of subrecords you want to update, pass in an episode if one exists.
```

Comment:
Probably worth adding Subrecord.bulk_update_from_dicts(parent, dicts, user) to subrecords.md as well?